indico/modules/search/client/js/components/SearchBox.jsx

```diff
@@ -23,33 +23,66 @@ const renderResult = ({label}, keyword) => (
   </div>
 );
 
-export default function SearchBox({onSearch, category}) {
+export default function SearchBox({onSearch, category, isAdmin}) {
   const [keyword, setKeyword] = useState('');
+  const [adminOverrideEnabled, setAdminOverrideEnabled] = useState(false);
   const options =
     category && !category.isRoot
       ? [
           {
             value: 'category',
-            title: category.title,
             label: Translate.string('In this category ↵'),
+            title: category.title,
+            renderer: props => renderResult(props, keyword),
+          },
+          {
+            value: 'global',
+            label: Translate.string('All Indico ↵'),
+            title: 'Global',
+            renderer: props => renderResult(props, keyword),
           },
-          {value: 'global', title: 'Global', label: Translate.string('All Indico ↵')},
         ]
       : [];
+  if (isAdmin) {
+    options.push({
+      value: 'global',
+      title: 'enableAdminOverride',
+      className: 'cursor-default',
+      onClick: () => {},
+      // eslint-disable-next-line react/display-name
+      renderer: () => (
+        <div styleName="option">
+          <Label content={Translate.string('ADMIN')} size="small" color="red" />
+          <Checkbox
+            styleName="checkbox-admin-search"
+            label={Translate.string('Skip access checks')}
+            checked={adminOverrideEnabled}
+            onChange={e => {
+              e.stopPropagation();
+              setAdminOverrideEnabled(!adminOverrideEnabled);
+            }}
+          />
+        </div>
+      ),
+    });
+  }
 
   const handleSearchChange = event => {
     setKeyword(event.target.value);
   };
 
-  // form submit happens when no option is selected in search (eg. in home category)
   const handleSubmit = () => {
     if (keyword.trim()) {
-      onSearch(keyword.trim(), category ? category.isRoot : true);
+      onSearch(keyword.trim(), category ? category.isRoot : true, adminOverrideEnabled);
     }
   };
 
   const handleResultSelect = (e, data) => {
-    onSearch(keyword.trim(), data.result.value === 'global');
+    if (data.result.title === 'enableAdminOverride') {
```

Comment:
can't we use the value for this? title is usually meant for humans, and while using the custom renderer never displays it, it feels wrong to use it like this..